app/views/listings/show.html.erb

```diff
@@ -47,6 +47,9 @@
                     <div role="tabpanel" class="tab-pane fade" id="tab-3">
                         <%= render 'gallery'%>
                     </div>
+                    <div role="tabpanel" class="tab-pane fade" id="tab-4">
+                        <%= render 'reviews'%>
```

Comment:
Average guest reviews must be computed.
Do not display each review like the following (not correct):
![image](https://user-images.githubusercontent.com/20994464/113800239-2dd24200-9789-11eb-82db-c54606df590c.png)